cpp/src/aztec/plonk/proof_system/constants.hpp

```diff
@@ -1,13 +1,13 @@
 #pragma once
 #include <cstdint>
 
-namespace waffle {
+namespace plonk {
 
 enum ComposerType {
     STANDARD,
     TURBO,
     PLOOKUP,
-    STANDARD_HONK,
+    STANDARD_HONK, // Todo(Arijit): We should replace STANDARD_HONK outside plonk or the namespace should be bonk
```

Comment:
This type is used to determine how the standard library constructs circuits using the given composer, and since we will be making a Honk composer that uses the `PLOOKUP` circuits, as well, it is better to make an enum `ComposerType` in `honk` and then move `STANDARD_HONK` into that enum and rename simply to `STANDARD`. That way we have `honk::ComposerType::Standard` and `plonk::ComposerType::Standard`.

Reply:
This makes sense. However, when I tried it, it seems like we need to compare between two different `enum` giving error ([-Wdeprecated-enum-compare](https://clang.llvm.org/docs/DiagnosticsReference.html#wdeprecated-enum-compare)). Instead, I created a constants file [here](https://github.com/AztecProtocol/barretenberg/blob/ad/remove-waffle/cpp/src/aztec/proof_system/constants.hpp#L1). Hope this resolves the issue.

Reply:
The ideal would be to make this work as Cody described above. If that's not possible (or if the scope of doing it is too large, etc.) I would prefer leaving `plonk/proof_system/constants.hpp` and `honk/proof_system/constants.hpp` in place (with `STANDARD_HONK ` inside of the of the plonk enum). That way we have the ugly `plonk::ComposerType::STANDARD_HONK` in one or two places rather than hundreds of e.g. `bonk::ComposerType::PLOOKUP` that will have to be reverted to `plonk::ComposerType::PLOOKUP` in the near future. I think this basically amounts to leaving things as you had them prior to the commit [4e37d5a](https://github.com/AztecProtocol/barretenberg/pull/140/commits/4e37d5ae33c1af9249fa426670c855594eb56830).

Reply:
Done as suggested.